fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/JunoOnboardingPageType.kt

```diff
@@ -6,9 +6,31 @@ package org.mozilla.fenix.onboarding.view
 
 /**
  * Model for different types of Onboarding Pages.
+ * @param sequencePosition Position of the page in the onboarding flow, used in telemetry.
+ * @param id Identifier for the page, used in telemetry.
  */
-enum class JunoOnboardingPageType {
-    DEFAULT_BROWSER,
-    SYNC_SIGN_IN,
-    NOTIFICATION_PERMISSION,
+enum class JunoOnboardingPageType(
+    val sequencePosition: String,
+    val id: String,
+) {
+    DEFAULT_BROWSER(
```

Comment:
Just a thought - would it be useful to have more detailed id's? e.g. `defaultBrowser, syncSignIn, notificationPermission` - then your comment example would be something like `defaultBrowser_syncSignIn`

Reply:
Gotcha 👍 I wondered if that was the case